packages/hint-strict-transport-security/src/hint.ts

```diff
@@ -198,6 +208,14 @@ export default class StrictTransportSecurityHint implements IHint {
                 } catch (err) {
                     // HTTPS site can't be fetched, do nothing.
                     debug(`${resource} doesn't support HTTPS`);
+
+                    /*
+                     * If the https resource can't be fetched, add
+                     * the domain to the unsupported list
+                     */
+                    if (err && err.error && err.error.code === 'ETIMEDOUT') {
```

Comment:
Can we receive other errors than `ETIMEDOUT`.
Maybe it's worth adding some tests to verify that we are not doing any more requests if the first time it fails and that way we can check if there are other type of errors coming from the server?

Reply:
>Can we receive other errors than ETIMEDOUT?

No, looks like some other erros can be received, like `ECONNRESET`.